subsys/net/l2/ethernet/arp.c

```diff
@@ -787,14 +787,6 @@ enum net_verdict net_arp_input(struct net_pkt *pkt,
 	struct net_pkt *reply;
 	struct in_addr *addr;
 
-	if (net_pkt_get_len(pkt) < (sizeof(struct net_arp_hdr) -
```

Comment:
Is it ok to drop the length check though? I think we should still verify the packet length before accessing the ARP header. It could be simplified to just `net_pkt_get_len(pkt) < (sizeof(struct net_arp_hdr)` I believe.

Reply:
I dropped it because it would not work any more, but I suppose we can add at least that ARP header len check there.